internal/providers/terraform/aws/testdata/kinesisanalyticsv2_application_test/kinesisanalyticsv2_application_test.golden

```diff
@@ -0,0 +1,21 @@
+
+ Name                                                Monthly Qty  Unit              Monthly Cost 
+                                                                                                 
+ aws_kinesisanalyticsv2_application.flink                                                        
+ ├─ Processing (stream)                                       10  KPU                      $1.27 
+ ├─ Processing (orchestration)                                 1  KPU                      $0.13 
```

Comment:
These costs aren't correct.  The quantities represent KPU "per month".   Since the prices in the db are for KPU-hours, the units need to be converted when calculating the cost.  

We basically want to keep the quantity/unit displayed as "10 KPU", since that is more understandable.  but the cost should be calculated as 10 * 730 * <KPU-hour price>.  Look at dyanmodb db `WCU` and `RCU` as an example of how this should be done using 
```
UnitMultiplier: schema.HourToMonthUnitMultiplier,
HourlyQuantity: decimalPtr(decimal.NewFromInt(writeCapacityUnits)),
```

Reply:
Done